lichess-bot.py

```diff
@@ -614,6 +716,7 @@ def should_exit_game(board: chess.Board, game: model.Game, prior_game: Optional[
 
 def final_queue_entries(control_queue: CONTROL_QUEUE_TYPE, correspondence_queue: CORRESPONDENCE_QUEUE_TYPE,
                         game: model.Game, is_correspondence: bool) -> None:
+    """Log the game that ended or we disconnected from, and sends a `local_game_done` for the game."""
```

Comment:
Add: If this is an unfinished correspondence game, put it in a queue to resume later.